lib/transbank/sdk/patpass/errors/integration_type_error.rb

```diff
@@ -0,0 +1,15 @@
+module Transbank
+  module Patpass
+    module PatpassByWebpay
+      module Errors
+        class IntegrationTypeError < ::Transbank::Patpass::Errors::PatpassError
+          attr_reader :code
+          def initialize(message, code)
```

Comment:
Code is not necessary, in other cases the code is included because those errors are related to HTTP requests. This one isn't, so you should eliminate that field

Reply:
Fixed!